doc/source/userguide.rst

```diff
@@ -120,7 +120,15 @@ size:
 3-D
 ---
 
-...
+For an input array of shape $(B, C, N_1, N_2, N_3)$, a Solid Harmonic Scattering with $J$
+scales and $L$ angular frequencies, which applies $P$ different types of $\mathcal L_p$ 
+spatial averaging will result in an output of shape
```

Comment:
2 points?

Reply:
What does that mean?

Reply:
shape <-> shape:  ?

Reply:
Francisms...

Reply:
In English, that's a “colon”.

Reply:
oh oops 🐶

Reply:
I disagree with putting a colon here. It's a math sentence, just with a newline.